docs/process/roadmap.md

```diff
@@ -0,0 +1,46 @@
+# ICU4X 1.0 Roadmap
+The following timeline is derived from [ICU4X 1.0 PRD](./prd.md) and is optimized to provide maximal alignment and synergy between progressive feature extension, technical advancements and business evaluations.
+
+* [x] 2020
+  * [x] October
+	  * [x] 0.1 Release
+  * [x] November/December
+	  * [x] Post 0.1 clean up and maintenance cycle
+* [ ] 2021
+	* [x] January/February
+		* [x] Locale
+			* [x] Likely Subtags
+		* [x] DateTime
+			* [x] Day Periods
+			* [x] Timezones
+			* [x] Components Bag
+		* [x] Unicode Set
+			* [x] L3a
+	* [ ] March/April
+		* [ ] FFI/WASM exploration
```

Comment:
I think this got moved to May (0.3)

Reply:
I assume it spans both since Manish and Shane are working on it in April, so I added it to both :)

Reply:
lmk if that works!